pkg/config/engine/crio/crio_test.go

```diff
@@ -144,3 +144,39 @@ func TestAddRuntime(t *testing.T) {
 		})
 	}
 }
+
+func TestGetRuntime(t *testing.T) {
+	logger, _ := testlog.NewNullLogger()
+
+	config := `
+[crio.image]
+signature_policy = "/etc/crio/policy.json"
+
+[crio.runtime]
+default_runtime = "crun"
+
+[crio.runtime.runtimes.crun]
+runtime_path = "/usr/libexec/crio/crun"
+runtime_root = "/run/crun"
+monitor_path = "/usr/libexec/crio/conmon"
+allowed_annotations = [
+    "io.containers.trace-syscall",
+]
+
+[crio.runtime.runtimes.runc]
+runtime_path = "/usr/libexec/crio/runc"
+runtime_root = "/run/runc"
+monitor_path = "/usr/libexec/crio/conmon"
+`
+	cfg, err := toml.Load(config)
+	require.NoError(t, err)
+
+	c := &Config{
+		Logger: logger,
+		Tree:   cfg,
+	}
+
+	rc, err := c.GetRuntimeConfig("crun")
```

Comment:
For containerd / crio / docker, can we add a test case for when the runtime name does not exist in the config?

Reply:
Done, please check now